meinberlin/apps/votes/templates/meinberlin_votes/token_export_dashboard.html

```diff
@@ -5,7 +5,8 @@
     <h1 class="u-first-heading">{% translate 'Download voting codes' %}</h1>
 
     <p class="form-hint u-bottom-divider">{% blocktranslate %}For voting in the third phase, all participants need their own voting code. Here you can download the codes in packages of {{ export_size }} per excel file. To generate the codes, please contact {{ contact_email }}.{% endblocktranslate %}</p>
-    <p class="u-bottom-divider">{% blocktranslate %}Number of generated codes: {{ number_of_module_tokens }}.{% endblocktranslate %}</p>
+    <p class="form-hint u-bottom-divider">{% blocktranslate %}Note that codes can only be downloaded once and then will be deleted from our database!{% endblocktranslate %}</p>
```

Comment:
proper wording & design still needed

Reply:
Maybe you could already add a screenshot to the story to find out where we should put the extra info?